superset/assets/src/explore/controls.jsx

```diff
@@ -814,7 +815,7 @@ export const controls = {
   druid_time_origin: {
     type: 'SelectControl',
     freeForm: true,
-    label: t('Origin'),
+    label: t(TIME_FILTER_LABELS.druid_time_origin),
```

Comment:
I think these should no longer be wrapped in `t()`

Reply:
fixed!